.github/workflows/pythonpublish.yaml

```diff
@@ -29,3 +29,10 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         ./scripts/publish
+        poetry run mkdocs build
+    - name: Push doc to Github Page
+      uses: peaceiris/actions-gh-pages@v2
+      env:
+        PERSONAL_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        PUBLISH_BRANCH: gh-pages
+        PUBLISH_DIR: ./site
```

Comment:
Just checking, but are you sure mkdocs build output goes to `./site`?

Reply:
Yep, the pages are generated in `./site` locally.